templates/includes/RequestItems_email.ss

```diff
@@ -0,0 +1,39 @@
+<table title="Requested documents" style="font-family: arial,sans-serif;width: 100%;border-collapse: collapse;">
+	<thead>
+	<tr>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;border-left: none;display: table-cell;">
+			<%t DMSDocument.TITLE "Title" %>
+		</th>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;display: table-cell;">
+			<%t DMSDocument.FILE_NAME "File name" %>
+		</th>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;display: table-cell;">
+			<%t DMSDocument.TOTAL_COPIES "# of copies" %>
+		</th>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;display: table-cell;">
+			<%t DMSDocument.TYPE "Type" %>
+		</th>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;display: table-cell;">
+			<%t DMSDocument.SIZE "Size" %>
+		</th>
+		<th style="padding: 0.8rem 1.3rem 0.6rem 1.3rem;background-color: #0085a0;font-family: arial,sans-serif;font-size: 1.8rem;color: #ffffff;font-weight: normal;text-align: left;vertical-align: top;display: table-cell;">
+			<%t DMSDocument.URL "URL" %>URL
+		</th>
+	</tr>
+	</thead>
+	<tbody>
+		<% loop $ItemsAsArrayList %>
+		<tr style="<% if Odd %>background-color: #ECF0F3;<% else %><% end_if %>">
+			<% with $Document %>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;border-left:none;">$Title</td>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;">$FilenameWithoutID</td>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;">$Up.Quantity</td>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;">$Extension.UpperCase</td>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;">$Size</td>
+				<td style="padding: 0.8rem 1.3rem;border: solid 0.1rem #BAC7D3;font-weight: normal;vertical-align: top;border-right:none;">
+					<a href="{$AbsoluteBaseURL}dmsdocument/{$ID}"><%t DMSDocument.VIEW_DOCUMENT "View Document" %></a></td>
+			<% end_with %>
+		</tr>
+		<% end_loop %>
+	</tbody>
+</table>
```

Comment:
To be honest @fspringveldt I think this would be better if you can use more semantic elements like `<dt>` and `<dd>`, spans etc rather than tables. I think using a table for the cart display itself is fine but the rest seems pretty 1990s